### PR TITLE
making menu more mobile friendly

### PR DESCRIPTION
**Pull request by readeral**
incomplete work as yet

### CODE REVIEW

File: components/nav-bar/nav-bar.tsx [hunk 1]
```diff
@@ -148,10 +143,10 @@ const NavBar: FC<NavBarProps> = ({menuItems}) => {
               }}
               onClick={toggleOpen}
             >
-              Menu
+              MENU
             </Button>
             <Can I="read" a="my-account">
-              <Link href="/my-account">My account</Link>
+              <Link href="/my-account">MY ACCOUNT</Link>
```
Comment (BarryThePenguin) [inline components/nav-bar/nav-bar.tsx]: We'll want to revert these and use `text-transform: uppercase;` instead

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

File: components/nav-bar/nav-items.tsx [hunk 2]
```diff
@@ -11,21 +11,25 @@ import NavLink from './nav-link';
 type NavCollapseProps = MenuItem;
 
 const NavCollapse: FC<NavCollapseProps> = ({text, childpages}) => {
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (BarryThePenguin) [inline components/nav-bar/nav-items.tsx]: It's pretty trivial, but there is a `use-toggle`

Comment (readeral) [reply components/nav-bar/nav-items.tsx]: Fixed

File: components/nav-bar/nav-bar.tsx (hunk 1 above)
Comment (readeral) [reply components/nav-bar/nav-bar.tsx]: Fixed

Comment (readeral) [reply components/nav-bar/nav-bar.tsx]: FWIW I know what text-transform is

**Pull request merged**